src/components/Error.astro

```diff
@@ -0,0 +1,96 @@
+                             @@@@@@@@@@@@@@@@@@@@@@@@@@@              @@@@@@@@@@@@              @@@@@@@@@@@@@                          @@@@@@@@@@@@@@@@@@@@@@@@@@@                                      
+                       @@@@@@@@@@@@@       @@@@@@@@@@@@@              @@@@@@@@@@@@                    @@@@@@@@@@@@               @@@@@@@@@@@@@       @@@@@@@@@@@@@                                      
+                       @@@@@@@@@@@@@       @@@@@@@@@@@@@              @@@@@@@@@@@@                    @@@@@@@@@@@@               @@@@@@@@@@@@@       @@@@@@@@@@@@@                                      
+                @@@@@@@@@@@@@@@@@@@@       @@@@@@@@@@@@@              @@@@@@@@@@@@                    @@@@@@@@@@@@        @@@@@@@@@@@@@@@@@@@@       @@@@@@@@@@@@@                                      
+                @@@@@@@@@@@@@              @@@@@@@@@@@@@              @@@@@@@@@@@@                    @@@@@@@@@@@@        @@@@@@@@@@@@@              @@@@@@@@@@@@@                                      
+                @@@@@@@@@@@@@              @@@@@@@@@@@@@              @@@@@@@@@@@@                    @@@@@@@@@@@@        @@@@@@@@@@@@@              @@@@@@@@@@@@@                                      
+                @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@        @@@@@@@@@@@@                    @@@@@@@@@@@@        @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@                                
+                @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@        @@@@@@@@@@@@                    @@@@@@@@@@@@        @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@                                
+                @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@              @@@@@@@@@@@@@             @@@@@@@@@@@@        @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@                                
+                                           @@@@@@@@@@@@@                    @@@@@@@@@@@@@             @@@@@@@                                        @@@@@@@@@@@@@                                      
+                                           @@@@@@@@@@@@@                    @@@@@@@@@@@@@             @@@@@@@                                        @@@@@@@@@@@@@                                      
+                                           @@@@@@@@@@@@@                          @@@@@@@@@@@@@@@@@@@@@                                              @@@@@@@@@@@@@                                      
+                                           @@@@@@@@@@@@@                          @@@@@@@@@@@@@@@@@@@@@                                              @@@@@@@@@@@@@                                      
+  </pre>
+  <h1 class="type-box" id="error-desc"></h1>
+</div>
+
+<script>
+  const typeMsg = () => {
+    typeIt("#error-desc", "\nThis incident will be reported", { speed: 8, mistakeProbability: 0 });
```

Comment:
Is this message even relevant? I don't see where this is going to be reported.

Reply:
This is a reference to [this](https://askubuntu.com/questions/641049/who-are-incidents-really-reported-to-and-how-can-a-sudo-user-access-the-reports)

Reply:
Doesnt make much sense to me.

Reply:
It is supposed to be something like "You tried to access a forbidden page. This incident will be reported."

Reply:
Funny toh hai. But technically it's not forbidden. Can we think of something else but equally quirky?